### PR TITLE
Created a new file for pgr_path_h structure

### PR DESCRIPTION
**Pull request by vidhan13j07**
Fixes https://github.com/cvvergara/pgrouting/issues/112.

Changes proposed in this pull request:
- Created a file for pgr_path_t.h
- Moved the structure pgr_path_t from pgr_types.h to pgr_path_t.h

### CODE REVIEW

**Comment by cvvergara** [comment]
@vidhan13j07 
Looks like that type is not used any more, Can you try without the file:

```
rm include/c_types/pgr_path_t.h
<compile locally>
```
if it works locally then do
```
git rm nclude/c_types/pgr_path_t.h
git push
```

if it does not work locally do:  (this will revert any changes)
```
git stash
```

**Comment by vidhan13j07** [comment]
@cvvergara The build was successful after removing `pgr_path_t.h`.

**Comment by cvvergara** [comment]
Cool one thing less :)

File: include/c_types/pgr_path_t.h [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/*PGR-GNU*****************************************************************
+File: pgr_types.h
```
Comment (vidhan13j07) [inline include/c_types/pgr_path_t.h]: I have removed this file algother in my next commit.

**Pull request merged**